lib/dpul_collections/indexing_pipeline/figgy/resource.ex

```diff
@@ -65,6 +72,23 @@ defmodule DpulCollections.IndexingPipeline.Figgy.Resource do
   end
 
   @spec to_map(resource :: %__MODULE__{}) :: map()
+  defp to_map(resource = %__MODULE__{internal_resource: "DeletionMarker"}) do
+    %{"resource_id" => [%{"id" => deleted_resource_id}], "resource_type" => [resource_type]} =
+      resource.metadata
+
+    # Create attributes specifically for deletion markers.
+    # 1. Replace existing metadata with a simple deleted => true kv pair
+    # 2. Set the entry id to the deleted resource's id
+    # 3. Set the entry internal_resource type to that of the deleted resource
+    resource
```

Comment:
Suggestion: maybe we can just explicitly create the map here? Unless there's still some properties we're saving.